packages/main/src/Avatar.hbs

```diff
@@ -18,7 +18,7 @@
 		{{/if}}
 
 		{{#if initials}}
-			<span class="ui5-avatar-initials">{{validInitials}}</span>
+			<span class="ui5-avatar-initials" aria-label="{{accessibleNameText}}">{{validInitials}}</span>
```

Comment:
This seems to work but I suggest that we set the aria-label on the root of the avatar and have it read out for every kind of avatar - with or without initials, with image or with icon. Also we can include this to the storybook sample and add tests.